app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextModuleActions.java

```diff
@@ -228,6 +232,24 @@ public void onClick(View view) {
                     });
                     return;
                 }
+                case "sort_todo": {
+                    SearchOrCustomTextDialogCreator.showSttSortDialogue(_activity, (callbackPayload) -> {
+                        ArrayList<SttTaskWithParserInfo> tasks = new ArrayList<>();
+                        for (String task : origText.split("\n")) tasks.add(sttcmd.parseTask(task));
+                        SttTaskComparator comparator = new SttTaskComparator();
+                        String[] split = callbackPayload.split(", ");
+                        comparator.setSortBy(split[0]);
+                        if (split[1].endsWith("ascending")) comparator.setAscending(true);
```

Comment:
comparator.setAscending(split[1]]........)

much easier and one line

Reply:
Fixed this in the updated branch to the suggested method.